addons/calendar/models/calendar_attendee.py

```diff
@@ -99,11 +99,17 @@ def _send_invitation_emails(self):
             self.env.ref('calendar.calendar_template_meeting_invitation', raise_if_not_found=False)
         )
 
-    def _send_mail_to_attendees(self, mail_template, force_send=False):
+    def _send_mail_to_attendees(self, mail_template, force_send=True):
```

Comment:
Don't really like changing the default to True to be honest. Keeping False would maybe mean to update all calls to it ?